src/main/java/teammates/ui/controller/InstructorCourseEditSaveAction.java

```diff
@@ -22,9 +22,7 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         InstructorAttributes instructor = logic.getInstructorForGoogleId(courseId, account.googleId);
         new GateKeeper().verifyAccessible(instructor, logic.getCourse(courseId), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE);
         
-        CourseAttributes courseToEdit = new CourseAttributes();
-        courseToEdit.id = courseId;
-        courseToEdit.name = courseName;
+        CourseAttributes courseToEdit = new CourseAttributes(courseId, courseName);
```

Comment:
If the attributes needs some 'cleaning', we should not make them public. Make them private and add getters/setters?